Tests/SonarQube.MSBuild.Tasks.UnitTests/IsTestFileByNameTests.cs

```diff
@@ -117,14 +117,9 @@ public void IsTestFile_InvalidRegexInConfig()
 
         [TestMethod]
         [TestCategory("IsTest")] // Regression test for bug http://jira.codehaus.org/browse/SONARMSBRU-11
-        public void IsTestFile_RetryIfConfigLocked()
```

Comment:
There were two tests covering reading the config file: one that tested what happened if the file read times-out, and one that tested if it didn't. I've moved both of these are to a new test class since they are no longer specific to the IsTestByName task.
However, we do still want to know that the task will fail if the read operation times out so I amended the IsTestFile_TimeoutIfConfigLocked test to check that case.